### ISSUE TITLE
Create tokens for components sizing

### ISSUE THREAD

**Issue by inesdgomes** [international-labour-organization/designsystem#707]
Just like we created tokens for spacing, should we do the same thing for sizing? Something to discuss before we action this task.

**Comment by beatrizmartinmartins** [comment]
Yes, we should totally do that. As of now I feel like the sizing tokens will be the same. Where will you include those tokens? I generally do it on icons only or do you asign them to height and widt for components? @justintemps

**Comment by beatrizmartinmartins** [comment]
@inesdgomes done. You can check it out here https://www.figma.com/file/qr8jQnxqJkRJFQnVC5PmrB/ILO-Foundations?type=design&node-id=1648%3A571&mode=design&t=8VBKSM4tjwjO5DX7-1

**Comment by justintemps** [comment]
Hey @inesdgomes and @beatrizmartinmartins, we have the tokens listed in [this file](https://github.com/international-labour-organization/designsystem/blob/develop/packages/themes/tokens/spacing/base.json)

This includes some legacy `padding-*` tokens which are being used by the new website so we didn't want to change or remove them.

**Comment by inesdgomes** [comment]
@beatrizmartinmartins Sorry for going back to this but if the sizes are exactly the same, wouldn't it make more sense to have one unique scale called 'Spacing and sizing'? I'm still forced to compare both to check if there are any differences

https://www.figma.com/design/qr8jQnxqJkRJFQnVC5PmrB/ILO-Foundations?node-id=1648-571&t=rq4iqSIaMCcywCxi-0

**Comment by beatrizmartinmartins** [comment]
Having the same group for spacing and sizing can be misleading. Imagine we want to change ONLY the sizes of the icons from 20 to 18, if the spacing and sizing group is the same, everything that has that 20 will change to 18, and we don't want to apply that change on the spacings, only on sizings. It's better to keep them separate to better differenciate between one another and have the possibility to change tokens in the future for certain things that don't necessarily impact other places. @inesdgomes hope this makes sense

**Comment by inesdgomes** [comment]
Agreed, thanks @beatrizmartinmartins.
@justintemps To be discussed if/when we want to implement the tokens for sizing in the DS.
